project.html

```diff
@@ -252,7 +261,7 @@ <h3>Object Modifications</h3>
             void main()
             {
                 if(currentTexture == 0){
-                    gl_FragColor =  mix(texture2D( defaultTex, fTexCoord ), fFogColor, fFogIntensity);
+                    gl_FragColor =  mix(texture2D(normalMap, normCoord)*texture2D( defaultTex, fTexCoord ), fFogColor, fFogIntensity);
```

Comment:
Not a normal mapping

Reply:
Please actually read the guide: https://learnopengl.com/Advanced-Lighting/Normal-Mapping